trikot-viewmodels-declarative/swift/swiftui/Components/VMDTextField.swift

```diff
@@ -12,13 +12,7 @@ public struct VMDTextField<Label>: View where Label: View {
         observableViewModel.viewModel
     }
 
-    private var text: Binding<String> {
-        Binding {
-            self.viewModel.text
-        } set: { text in
-            self.viewModel.onValueChange(text: self.viewModel.formatText(text))
-        }
-    }
+    @State private var text: String = ""
```

Comment:
Really not sure about that change, any text transformation should be handled by the view model anyway.